consensus/types/src/chain_spec.rs

```diff
@@ -496,6 +495,16 @@ impl ChainSpec {
         Hash256::from(domain)
     }
 
+    pub fn compute_activation_exit_epoch(&self, epoch: Epoch) -> Result<Epoch, ArithError> {
```

Comment:
probably worth putting a comment above this that links to the URL of the spec function